server/jetstream_test.go

```diff
@@ -17508,6 +17508,25 @@ func TestJetStreamWorkQueueSourceNamingRestart(t *testing.T) {
 	}
 }
 
+func TestJetStreamDisabledHealthz(t *testing.T) {
+	s := RunRandClientPortServer()
+	if config := s.JetStreamConfig(); config != nil {
+		defer removeDir(t, config.StoreDir)
+	}
+	defer s.Shutdown()
+
+	if s.JetStreamEnabled() {
+		t.Fatalf("Expected JetStream to be disbaled")
```

Comment:
Typo here that causes our checks to fail:
```
./server/jetstream_test.go:17518:37: "disbaled" is a misspelling of "disabled"
```

Reply:
fixed